userguide/dailyuse/waydroid.rst

```diff
@@ -0,0 +1,81 @@
+Android apps
+========================
+
+`Waydroid <https://waydro.id>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
+
+.. note::
+    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
+    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
+
+.. note::
+    You will need to execute commands on your Ubuntu Touch device (and/or attached computer) to initialize Waydroid and install APKs.
+    You can do that by using the terminal application, but it is easier to use ``adb shell`` or :doc:`set up SSH </userguide/advanceduse/ssh>` to access your Ubuntu Touch device from your computer.
+
+
+How to set up
+-------------
+
+All Halium 9 and newer devices ship with waydroid out of the box starting Ubuntu Touch focal, if your device fits these requirements you can follow the folowing steps
+
+#. Run ``adb shell`` from your computer to get a shell on your Ubuntu Touch device (or open the terminal app on your ubuntu touch device).
+#. Run the following command on your Ubuntu Touch device: ``waydroid init``.
+
+Now you're done! You might have to refresh the app drawer (pull down from the center of the screen and release) for the new Android apps to show up.
+
+How to install new APKs
+-----------------------
+
+- Copy the APK to ``/home/phablet/Downloads``. Then run the following from your computer::
```

Comment:
I'd suggest to add a separate bullet point, for the "open a shell....", I think it's best to just copy paste from above, you know keep the same pattern, makes it easier to read. and then a separate bullet point for waydroid app install. No need for exit I think. Again keep the same pattern as above

Reply:
ah. also, please pick either enumeration or itemization and use consistently for both lists. And either make the "Done... refresh .. pull" sentence in both cases part of the list or not part of the list

Reply:
made it more consistent